README.md

```diff
@@ -58,84 +58,131 @@ Here is an overview of all fields that can be present in plugin meta YAML files
 
 |field name in YAML | description | mandatory |
 |-------|-------|-------|
-| id | plugin ID | yes |
+| publisher | publisher name | yes |
+| deprecate.migrateTo | which plugin should be used instead when plugin is deprecated | no |
```

Comment:
what is the format? `publisher`/`name`/`version`?

Reply:
yes

Reply:
added